src/component/RoadmapCanvas.js

```diff
@@ -34,35 +35,34 @@ export class RoadmapCanvas {
    * @param {Roadmap} roadmap
    */
   render(roadmap) {
-    if (this.type === "story") {
-      this._render(roadmap, roadmap.getSortedStoryList());
-    } else if (this.type === "assignee") {
-      this._render(roadmap, roadmap.getSortedAssigneeList());
+    // TODO: check. Is this condition need?
```

Comment:
Maybe *I* will not use `all` type (display  based on `sotry` and `assignee` ). So I think that it's ok to delete this condition.
But if *someone* wants to use `all` type, I can't delete this condition.
Or, refactor `Roadmap` and call `draw` on twice time.

Now
```javascript
let roadmap = new Roadmap.constructor(options, dataSeta);
roadmap.draw();
```
After
```javascript
let roadmap = new Roadmap.constructor(dataSeta);
roadmap.draw(options);
roadmap.draw(options);
```